tasks/managed/rh-sign-image/rh-sign-image.yaml

```diff
@@ -47,7 +47,7 @@ spec:
         size of batch attributes to send to internal-request. As internal request arguments are need to be
         strings, size here represent maximal string length of `references` and `manifest_digests` sent to
         internal request
-      default: 4096
+      default: 16384
```

Comment:
I don't think you can do this. As the description reads, this is already at the max limit allowed

Reply:
4096 is the max character limit for tekton results. Is it not for parameters too?

Reply:
That is the maximum character limit for Tekton results because results are stored in termination messages (https://github.com/tektoncd/pipeline/issues/4808). Parameters don't have this limitation.